sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -292,6 +296,13 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     };
   }
 
+  getLoggingIdentifier(): boolean | undefined{
```

Comment:
should need some formatting here

Reply:
We should also probably make those internal as they're not for public usage

Reply:
removed this function